### PR TITLE
feat: ingest-replay-events schema

### PR DESCRIPTION
**Pull request by lynnagara**
I generated a schema for ingest-replay-events based on data I observed on this topic in prod.

Though this looks entirely different from the schema in https://github.com/getsentry/sentry-kafka-schemas/pull/143/

### CODE REVIEW

File: schemas/ingest-replay-events.v1.schema.json [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$ref": "#/definitions/ReplayEvent",
+  "title": "replay_event",
+  "definitions": {
+    "ReplayEvent": {
+      "type": "object",
+      "properties": {
+        "type": {
+          "type": "string",
+          "enum": ["something", "replay_event"]
```
Comment (untitaker) [inline schemas/ingest-replay-events.v1.schema.json:11]: This looks kinda wrong. are we really sending type=something in prod?

Comment (lynnagara) [reply schemas/ingest-replay-events.v1.schema.json:11]: oh shit, oops

Comment (lynnagara) [reply schemas/ingest-replay-events.v1.schema.json:11]: i hand wrote that

Comment (lynnagara) [reply schemas/ingest-replay-events.v1.schema.json]: Updated. I only actually saw `replay_event` in prod

**Review by untitaker — APPROVED** [review]
it lgtm but would like replays team to take a look

**Review by JoshFerge — APPROVED** [review]
makes sense to me.

**Pull request merged**